.github/workflows/plugin-release.yml

```
@@ -0,0 +1,200 @@
+      run: dotnet restore
+    
+    - name: Build project
+      run: dotnet build --configuration Release --no-restore
+    
+    - name: Get modified files
+      id: get_changes
+      run: |
+        $plugins = @()
+        $legalityChanged = $false
+        $changedFiles = git diff --name-only HEAD~1 HEAD
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)